lib/mssql.js

```diff
@@ -124,6 +124,9 @@ MsSQL.prototype.executeSQL = function(sql, params, options, callback) {
   }
   var innerCB = function(err, data) {
     debug('Result: %j %j', err, data);
+    if (data && data[0] && data[0].affectedRows !== undefined && data[0].affectedRows !== null) {
```

Comment:
I think `count` is the property. In our case, since we need to unify the `affectedRows` attribute amongst connectors, so I used it here: `SELECT @@ROWCOUNT as affectedRows'`